Example/EpoxyExample/ViewControllers/LayoutGroups/TextRowExampleViewController.swift

```diff
@@ -0,0 +1,48 @@
+// Created by Tyler Hedrick on 1/26/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import EpoxyCollectionView
+import UIKit
+
+class TextRowExampleViewController: CollectionViewController {
+
+  // MARK: Lifecycle
+
+  init() {
+    super.init(layout: UICollectionViewCompositionalLayout.list)
+    setItems(items, animated: false)
+  }
+
+  // MARK: Internal
+
+  override var title: String? {
+    get { "Label alignment" }
+    set { }
+  }
+
+  var items: [ItemModeling] {
```

Comment:
nit: `@ItemModelBuilder` to remove nested `[]`
```suggestion
  @ItemModelBuilder
  var items: [ItemModeling] {
```